src/components/general/FilterPane/index.tsx

```diff
@@ -15,7 +15,7 @@ import applyFilters, {
 import { applyCountAsync, Count, FilterCount } from './countFilters';
 
 export { default as FilterTypes } from './filterTypes';
-export { applyFilters, FilterTerm, FilterSection, Count, FilterCount };
+export { applyFilters, FilterTerm, FilterSection, Count, FilterCount, CollapseExpandButton };
```

Comment:
CollapseExpandButton should probably be moved to a seperate component? Or reimplemented in navigation drawer

Reply:
Reimplemented